content/F2015/F15-event-career-panel.md

```diff
@@ -0,0 +1,21 @@
+title: CSC and WiCS Career Panel
+Date: 2015-09-24 17:00-19:00
```

Comment:
Possible clarification: date is the posting date, not the event date. I deleted the date range in Fatema's last post when I reformatted it; I don't know if this actually parses. 

I would suggest setting this to today's date, and possibly adding an "Update" date later (or just changing the date) to keep the Orientation post as our featured post on the front page. Does that make sense?